package.json

```diff
@@ -39,22 +39,23 @@
     "ts-results": "^3.3.0"
   },
   "peerDependencies": {
-    "casper-js-sdk": "^2.10.0"
+    "casper-js-sdk": "^2.12.0"
   },
   "devDependencies": {
-    "casper-js-sdk": "^2.10.0",
     "@types/jest": "^29.4.0",
     "@typescript-eslint/eslint-plugin": "^5.53.0",
     "@typescript-eslint/parser": "^5.53.0",
     "browserify-zlib": "^0.2.0",
     "buffer": "^6.0.3",
+    "casper-js-sdk": "^2.12.0",
```

Comment:
pls specify `2.11.0` as a minimum version here, because version >=2.12.0 has some problems with earlier versions of node

Reply:
I have a question, do we need casper-js-sdk in devDependency? @ihor

Reply:
@gyroflaw , pls
1) Install packages using nodejs version from `.nvmrc` file and npm version `8.19.2`, because as I see there are too many changes in `package-lock.json` file
2) Specify `casper-js-sdk` version  `^2.11.0` in `package.json`, because we had problems with higher versions on our side, you might still higher version in your project

Reply:
> Install packages using nodejs version from .nvmrc file and npm version 8.19.2, because as I see there are too many changes in package-lock.json file

I will do

> Specify casper-js-sdk version ^2.11.0 in package.json, because we had problems with higher versions on our side, you might still higher version in your project

Could you please let me know what problems you had with higher versions? @Vladimir-Kuchinskiy

Reply:
@gyroflaw sure, I noticed that we do not have a problem with `casper-js-sdk@2.12.0` but we do have it for `casper-js-sdk@^2.13.0`, so you can left it as is.
When we used `casper-js-sdk@^2.13.0` our event handler was failing to startup with this error.
```
@jsonMember on e.StoredValue: cannot resolve specified property constructor at runtime. If the type is not yet defined, for example due to circular references, add '() => ' before it. E.g. @jsonMember(() => Foo)
Error: Cannot find module 'node:crypto'
```
Currently we use node 12 for our event handler on clarity project, and we are not going to upgrade that version in a nearest future because we have plans to switch to a newer golang version of the event handler instead.

Reply:
I suppose it is because of the version of node 12 does not have this package in its standard library, but it is only my suspect and we can extract this conversation out of this PR, so we do not block the next release of the library